go/vumitools/account/models.py

```diff
@@ -26,10 +26,23 @@ class UserAppPermission(Model):
     application = Unicode(max_length=255)
 
 
+def flag_method(name):
+    def fget(self):
+        return name in self.flags
+
+    def fset(self, value):
+        if value:
+            self.flags.append(name)
+        else:
+            del self.flags[list(self.flags).index(name)]
```

Comment:
I'll add `.remove()` to `ListOf` in praekelt/vumi#892, since it makes sense to have it.